sympy/matrices/common.py

```diff
@@ -2018,6 +2018,75 @@ def replace(self, F, G, map=False, simultaneous=True, exact=None):
         """
         return self.applyfunc(
             lambda x: x.replace(F, G, map=map, simultaneous=simultaneous, exact=exact))
+    
+    def rot90(self, k = 0):
```

Comment:
In PEP8 convention, you should not have spacing around `=` for optional parameters
And you should set default `k` to `1`. 
```suggestion
    def rot90(self, k=1):
```

Reply:
Alrighty 😄 ...

Reply:
It also applies for examples